package-system/tiff/build_tiff_android.cmd

```diff
@@ -5,14 +5,28 @@
 @rem # SPDX-License-Identifier: Apache-2.0 OR MIT
 @rem #
 
-@IF "%LY_ANDROID_NDK_ROOT%" == "" (
-    @echo Need to set the varible LY_ANDROID_NDK_ROOT to the location of your NDK install!
-    @exit /b 1
-)
+@rem # note that we explicitly turn off the compilation of all features that rely on 3rd Party Libraries
+@rem # except the ones we want.  This prevents the cmake build system from automatically finding things
+@rem # if they happen to be installed locally, which we don't want.
+cmake -S temp/src -B temp/build -G Ninja ^
```

Comment:
Problem here that I've seen with double-linking static libs on Android and other platforms - if a build product links its own libz somewhere (very likely) then this could result in build incompatibilities and problems resolving symbols in a monolithic binary. Does not need to be addressed here but we may require some advanced user documentation on which 3P libraries are built by O3DE, what their default configurations are, and where to make modifications to builds.

Maybe things have changed but I'm curious if this remains an issue on static-link platforms under certain compilers.

Reply:
Yes.  Of course, this new zlib package is actually compatible with those other libraries.  So for example, I've already tested it but tifflib, as well as openexr, openimageio, and python can all use the O3DE Zlib as a drop in replacement.  
Not to be addressed in this, of course.  But going forward if we do have a 3p package that satisifies a dep, I'd prefer if we compile using that instead of a copy.